liveRecorder/Tasks/UploadTask.py

```diff
@@ -93,6 +93,43 @@ def append_recording_handler(self, file_full_path, flavor_id, is_first_flavor):
         self.backend_client.set_recorded_content_local(partner_id, self.entry_id, file_full_path,
                                                        str(float(self.duration)/1000), self.recorded_id, flavor_id)
 
+    def update_recording_status(self):
+        try:
+            with open(self.metadata_full_path, "r") as dc_file:
+                data = dc_file.read()
+                if data:
+                    metadata = json.loads(data)
+                    entry_server_node_id = -1
+                    try:
+                        entry_server_node_id = int(metadata['entryServerNodeId'])
+                        self.logger.debug(
```

Comment:
why do we need it if you are doing get get_server_entry_nodes_list anyways?
you will get only two ids in the response and the you know if the serverNode is on same DC...

Reply:
The motivation for the status is to avoid processing job if stream is live. such as in following use case:
network throttling caused recording session to end in DC#1 but stream is still live in DC#2. According to Yossi, if I upload the VOD from DC#1 while DC#2 is live it can cause problem. So I use it mainly to verify that I don't process and upload VOD while the stream is live and recording on same RecordedEntryId